jobs/subseasonal/prep/JEVS_SUBSEASONAL_PREP

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -xa
+
+# ###################################
+# SET SHELL PROCESSING VARIABLES
+# ###################################
+export PS4='$SECONDS + ' 
+date
+
+export machine="WCOSS2"
+
+###########################################################
+# obtain unique process id (pid) and make temp directories
+###########################################################
+export pid=$$
+#export DATA_IN=${DATA_IN:-$DATAROOT}
+export DATA_IN=${DATA_IN:-$DATAROOTtmp}
+export DATA=$DATA_IN/${pid}
+
+
+#rm -rf $DATA_IN
+mkdir -p $DATA
+cd $DATA 
+
+###############################
+# Determine Job Output Name on System
+###############################
+export pgmout="OUTPUT.${pid}" 
+
+################################################################
+# SENDCOM=YES--Copy files from TMPDIR to $COMOUT
+# SENDECF=YES--Flag events on ecflow
+# SENDDBN=YES--Issue DBNet client calls
+################################################################
+export SENDCOM=${SENDCOM:-NO}
+export SENDDBN=${SENDDBN:-NO}       # need to set to NO for testing
+export SENDECF=${SENDECF:-NO}
+export SENDDBN_NTC=${SENDDBN_NTC:-NO}
+################################################################
+# Specify Execution Areas
+################################################################
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
+export EXECevs=${EXECevs:-$HOMEevs/exec}
+export PARMevs=${PARMevs:-$HOMEevs/parm}
+export USHevs=${USHevs:-$HOMEevs/ush}
+export FIXevs=${FIXevs:-/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix}
```

Comment:
Change line 47 to:
`export FIXevs=${FIXevs:-$HOMEevs/fix}`
In the ecf script define the override setting for $FIXevs.